cli/ops/fs.rs

```diff
@@ -611,7 +618,7 @@ fn op_utime(
   let is_sync = args.promise_id.is_none();
   blocking_json(is_sync, move || {
     debug!("op_utime {} {} {}", args.filename, args.atime, args.mtime);
-    utime::set_file_times(args.filename, args.atime, args.mtime)?;
+    deno_fs::set_file_times(args.filename, args.atime, args.mtime)?;
```

Comment:
Unnecessary change of import

Reply:
Let me explain the thinking that prompted this, it'd help to see what mistake I was making. Initially, I was confused what was the point of many operations being split between `cli/ops/fs.rs` and `cli/fs.rs`. For example, why is the function `chown` in `cli/fs.rs` rather than just being folded into the `op_chown` in `cli/ops/fs.rs`. (It's not invoked from anywhere else.) Similarly for the `mkdir` and `make_temp` code in `cli/fs.rs`.

I would have asked someone if I knew what was a good channel to do that in, but I didn't. Maybe the gitter lobby is the place to do that, but there's a lot of chatter there.

Anyway, as I was working on adding more filesystem ops, and expanding the functionality of some existing ones, I gradually got the impression that oh, the point of `cli/fs.rs` is to consolidate all of the call-outs to third-party crates (including `nix`, but not `tokio`), and maybe also the call-outs to `unix::fs` stuff. The existing codebase seemed to match that impression, but only 95%. There were still these two calls to `remove_dir_all` and `set_file_times` in `cli/ops/fs.rs`. So I thought, Ok, let's just make sure they're exported from `cli/fs.rs` (as `pub use`s), and have `cli/ops/fs.rs` import them from there. Hence this commit.

I agree that looked at from another angle, this change is pointless. But then I'd value some guidance in better understanding when new functionality should be partly placed in `cli/fs.rs`, as you guys did for `chown` etc, and what should be wholly included in `cli/ops/fs.rs`.

Reply:
> For example, why is the function chown in cli/fs.rs rather than just being folded into the op_chown in cli/ops/fs.rs.

Functionality should be folded into the ops themselves unless it's being used elsewhere. We have gone through many refactors, s things might be a certain way just from oversight rather than by design. I think that's the situation with chown.

Reply:
OK, I'll undo the attempts to consolidate the third-party imports into `cli/fs.rs`. For the moment I'll leave `chown` where it is --- and maybe I'll push some of the other stuff I've got in this sequence, which as it stands puts third-party stuff into `cli/fs.rs`. But at some point soon, once more of this stuff has worked its way into the codebase, I'll move `chown` etc (and anything I add to `cli/fs.rs`) back into `cli/ops/fs.rs`.